unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmotesHUD/EmotesHUDController.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using DCL;
+using UnityEngine;
+
+//TODO rename to EmotesHUDController
```

Comment:
As the class name is already "EmotesHUDController"... should we remove this TODO comment?